src/lib/y2network/connection_config/lcs.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/connection_config/base"
+
+module Y2Network
+  module ConnectionConfig
+    # Configuration for lcs connections
+    class Lcs < Base
+      # @return [String] read bus id
+      attr_accessor :read_channel
+      # @return [String] write bus id
+      attr_accessor :write_channel
+      # @return [Integer] connection protocol
+      attr_accessor :protocol
+      # @return [Integer] lcs timeout
+      attr_accessor :timeout
+
+      # Constructor
+      def initialize
+        super()
+        @protocol = 0
+        @timeout = 5
```

Comment:
rationale for this number?

Reply:
Use current default from LanItems, but will also document with official driver DEFAULT value link:

https://github.com/torvalds/linux/blob/6f0d349d922ba44e4348a17a78ea51b7135965b1/drivers/s390/net/lcs.h#L122